convert-config/entrypoint.sh

```diff
@@ -16,6 +16,7 @@ if [ -z "${INPUT_BRANCH_NAME}" ]; then
    export INPUT_BRANCH_NAME=main
 fi
 
+git config --global --add safe.directory "/github/workspace"
```

Comment:
Just a nit, but can we reuse `WORKSPACE` from here? - https://github.com/RedHatInsights/rbac-config-actions/pull/11/files#diff-611abac04bd322077ca4dd304c5fa21167e3065ed0fe4ba1caee7e429547640fR3

Reply:
Yes! Updated 👍